src/airflow/airqo_etl_utils/kcca_utils.py

```diff
@@ -320,7 +319,7 @@ def transform_kcca_hourly_data_for_bigquery(data: list) -> list:
         device_name = data_row["deviceCode"]
         site_id, _ = get_site_and_device_id(devices, device_name=device_name)
         if not site_id:
-            continue
+            site_id = ""
```

Comment:
defaulting site id to am empty string for KCCA devices without sites

Reply:
This could have some significance in the ongoing discussion about access of data for undeployed devices. cc: @sserurich 

Thanks @NoahNsimbe